lib/src/util.dart

```diff
@@ -1307,18 +1307,20 @@ void svg_to_png_data() {
 
   // Translates dna_sequence_element_copy down so that it's Blob uri captures the topmost dna sequence
   // inside of it's view box.
+  Rect bbox = dna_sequence_element.getBBox();
```

Comment:
Can you double-check that the function `svg_to_png_data()` (which calls `document.getElementById` above) is never being called in either a `render()` method of a React component or in a reducer? Both of those should be pure functions (of their parameters/props) and not directly reference "global" information such as DOM elements returned from `document.getElementById`.

Reply:
Only place it is called is here
https://github.com/UC-Davis-molecular-computing/scadnano/blob/dc1e16f3209f71239fff5a672b2b5101f4426927/web/index.html#L284

Attached as a callback to the `main_view_svg_pan_zoom` object here:
https://github.com/UC-Davis-molecular-computing/scadnano/blob/dc1e16f3209f71239fff5a672b2b5101f4426927/web/index.html#L328